Packages/Signer/Sources/Chains/TronSigner.swift

```diff
@@ -133,9 +133,32 @@ public struct TronSigner: Signable {
             $0.data = data
             $0.callValue = callValue
         }
-        return try [
-            sign(input: input, contract: .triggerSmartContract(contract), feeLimit: Int(feeLimit), privateKey: privateKey)
-        ]
+
+        if let approval = quoteData.approval {
+            let spender = WalletCore.Base58.decodeNoCheck(string: approval.spender)?.dropFirst() ?? Data()
```

Comment:
```suggestion
            let spender = WalletCore.Base58.decodeNoCheck(string: approval.spender)?.dropFirst() ?? Data()
```

should avoid doing `?? Data()`, use try